app/forms.py

```diff
@@ -0,0 +1,45 @@
+from django import forms
+from app.models.user import User
+from django.core.exceptions import ValidationError
+from django.contrib.auth import authenticate
+
+
+class SignUpForm(forms.ModelForm):
+    email = forms.EmailField(required=True)
+    password = forms.CharField(widget=forms.PasswordInput)
+    class Meta:
+        model = User
+        fields = ('email', 'passwrod')
+        widgets = {
+            'email' : forms.TextInput(attrs={'class': 'form_control'}),
+            'password' : forms.TextInput(attrs={'class': 'form_control'}),
+        }
+    def verify_email(self):
```

Comment:
it's better to use clean_fieldname() as method name,
it will be automatically called in  full_clean method 
![image](https://github.com/user-attachments/assets/2bed0c62-a10c-4497-a76d-c470e843bc99)